### PR TITLE
CMS-1792 Allow international article pages as child pages of international sectors

### PR DESCRIPTION
**Pull request by sdonk**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: great_international/serializers.py [hunk 1]
```diff
@@ -657,6 +657,7 @@ class InternationalSectorPageSerializer(
     ChildPagesSerializerHelper
 ):
     child_sub_sectors = serializers.SerializerMethodField()
+    child_sub_articles = serializers.SerializerMethodField()
```
Comment (eadpearce) [inline great_international/serializers.py]: can this just be `articles` or `child_articles` pls

File: great_international/serializers.py [hunk 2]
```diff
@@ -665,6 +666,13 @@ def get_child_sub_sectors(self, obj):
             MinimalPageSerializer
         )
 
+    def get_child_sub_articles(self, obj):
```
Comment (MichaelWalker) [inline great_international/serializers.py]: Do we also need to add a subscription for the cache busting? (or does this already work?)

Comment (sdonk) [reply great_international/serializers.py]: Forgot about that, good catch!

Comment (sdonk) [reply great_international/serializers.py]: Somehow it's already there https://github.com/uktrade/directory-cms/blob/develop/great_international/cache.py#L14 @MichaelWalker

**Pull request merged**